app/src/main/java/org/mozilla/focus/locale/screen/LanguageFragment.kt

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.locale.screen
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.platform.ComposeView
+import kotlinx.coroutines.launch
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.lib.state.ext.observeAsComposableState
+import mozilla.components.support.locale.LocaleUseCases
+import org.mozilla.focus.R
+import org.mozilla.focus.components.StoreProvider
+import org.mozilla.focus.ext.components
+import org.mozilla.focus.ext.requireComponents
+import org.mozilla.focus.settings.BaseSettingsComposeFragment
+import org.mozilla.focus.state.AppAction
+import org.mozilla.focus.state.Screen
+
+class LanguageFragment : BaseSettingsComposeFragment() {
+    private lateinit var browserStore: BrowserStore
+    private lateinit var localeUseCase: LocaleUseCases
+    private lateinit var languageScreenStore: LanguageScreenStore
+    private lateinit var screenInteractorDefault: DefaultLanguageScreenInteractor
```

Comment:
Instead of `lateinit`, could some of them be initialized from a `lazy { .. } ` block? `lateinit` has bitten us a couple of times. Code gets refactored/moved and suddenly there are edge cases where non initialized properties get accessed. I try to avoid it whenever possible.